doc/speed-manual.tex

```diff
@@ -219,68 +219,111 @@ \section{Job Management}
+the scheduler commands. 
+For example, if, \texttt{qstat -f -u "*"}, returns something non-error related, 
+you are in business. 
 
-That command prepares your environment (again, be careful overwriting an existing
-\file{.tcshrc}), provides you with a job template, and sets up your
-cluster-specific storage. Note that you need to either log out and back
-in, or execute a new shell, for the environment changes to be applied
-(important). And if you use \tool{bash}, or another shell, please contact 
-\texttt{rt-ex-hpc AT encs.concordia.ca}.
+The next step is to copy a job template to your home directory and to set up your
+cluster-specific storage. Execute the following command from within your
+home directory. (To move to your home directory, type \texttt{cd} at the Linux
+prompt and press \texttt{Enter}.) 
 
-If you are connecting from home, and have a Mac or Linux system, in a
-terminal, this will connect you (on a single line):
+\begin{verbatim}
+cp /home/n/nul-uge/template.sh . && mkdir /speed-scratch/$USER
+\end{verbatim}
+
+\textbf{Tip:} Add the source command to your shell-startup environment. 
+
+For \textbf{new ENCS Users}, and/ or those who don't have an \file{.tcshrc} 
+file, move a copy of the \file{.tcshrc} in \texttt{nul-uge}'s. home directory 
+to your home directory with:
 
 \begin{verbatim}
-ssh -o ProxyCommand="ssh <ENCSusername>@login.encs.concordia.ca nc speed 22"
-  <ENCSusername>@speed.encs.concordia.ca
+cp /home/n/nul-uge/.tcshrc . 
 \end{verbatim}
 
-Windows users can connect via PuTTY (or MobaXterm). All users are expected
-to have a basic understanding of Linux and its commonly used commands.
+Users who already have an \file{.tcshrc} file, use a text editor, such as
+ \tool{vim} or \tool{emacs}, to add the source request to your existing
+shell-startup environment (i.e., to the \file{.tcshrc} file in your home directory). 
+
+Note that you will need to either log out and back in, or execute a new shell, 
+for the environment changes in the updated \file{.tcshrc} file to be applied 
+(important). 
+
+If you use \tool{bash}, or another shell, please contact 
+\texttt{rt-ex-hpc AT encs.concordia.ca}.
 
 % ------------------------------------------------------------------------------
 \subsection{Job Submission Basics}
 
-Preparing your job for submission is fairly straightforward. Job scripts
-are broken into four main sections: make a copy of \file{template.sh}
-or any existing job script example on our GitHub, and base
-your job on that. The first section is the shell call, and that can be
-left alone, unless you want to use a different shell. The next section
-contains the options provided to the cluster scheduler, and \file{template.sh}
-provides the essentials:
+Preparing your job for submission is fairly straightforward. Editing a copy
+of the\file{template.sh} you moved into your home directory during section
+\refname{envsetup} is a good place to start. You can also use a job script
```

Comment:
Missing space before \file and instead of \refname please use \xs

Reply:
Nice catch. Will use \xs going forward